src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -60,62 +91,202 @@ export class PaymentRoute extends Component<PaymentProps, PaymentState> {
-                      </>
+                      {errorMessage && (
+                        <Sans pt={1} size="2" color="red100">
+                          {errorMessage}
+                        </Sans>
+                      )}
+                    </Flex>
+                    <Checkbox
+                      selected={this.state.hideBillingAddress}
+                      onSelect={hideBillingAddress =>
+                        this.setState({ hideBillingAddress })
+                      }
+                    >
+                      Billing and shipping addresses are the same
+                    </Checkbox>
+                    <Collapse open={!this.state.hideBillingAddress}>
+                      <AddressForm
+                        defaultValue={this.state.address}
+                        onChange={address => this.setState({ address })}
+                        billing
+                      />
+                    </Collapse>
+                    {!xs && (
+                      <ContinueButton
+                        onClick={this.onContinueButtonPressed}
+                        loading={isComittingMutation}
+                      />
                     )}
-                  </Flex>
-                }
-              />
-            )}
-          </Responsive>
-        </Elements>
+                  </Join>
+                  <Spacer mb={3} />
+                </>
+              }
+              Sidebar={
+                <Flex flexDirection="column">
+                  <TransactionSummary order={order} mb={[2, 3]} />
+                  <Helper
+                    artworkId={order.lineItems.edges[0].node.artwork.id}
+                  />
+                  {xs && (
+                    <>
+                      <Spacer mb={3} />
+                      <ContinueButton
+                        onClick={this.onContinueButtonPressed}
+                        loading={isComittingMutation}
+                      />
+                    </>
+                  )}
+                </Flex>
+              }
+            />
+          )}
+        </Responsive>
       </>
     )
   }
+
+  private getSelectedBillingAddress(): stripe.TokenOptions {
+    const {
+      name,
+      addressLine1,
+      addressLine2,
+      city,
+      region,
+      postalCode,
+      country,
+    } = (this.state.hideBillingAddress
+      ? this.props.order.requestedFulfillment
+      : this.state.address) as Address
+
+    return {
+      name,
+      address_line1: addressLine1,
+      address_line2: addressLine2,
+      address_city: city,
+      address_state: region,
+      address_zip: postalCode,
+      address_country: country,
+    }
+  }
+
+  private createCreditCard({ token }) {
+    commitMutation(this.props.relay.environment, {
+      onCompleted: (data, errors) => {
+        if (data && data.createCreditCard.creditCardOrError.creditCard) {
+          console.log("successfully added credit card to Gravity", data)
```

Comment:
Not sure if we want to leave this in here either

Reply:
👍 this should be removed.